packages/grafana-ui/src/components/BigValue/BigValue.story.tsx

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import { color, number, select, text } from '@storybook/addon-knobs';
-import { BigValue, BigValueColorMode, BigValueGraphMode, BigValueJustifyMode, BigValueTextMode } from './BigValue';
+import { Story } from '@storybook/react';
+import {
+  BigValue,
+  BigValueColorMode,
+  BigValueGraphMode,
+  BigValueJustifyMode,
+  BigValueTextMode,
+  Props,
+} from './BigValue';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import mdx from './BigValue.mdx';
 import { useTheme } from '../../themes';
 import { ArrayVector, FieldSparkline, FieldType } from '@grafana/data';
 
-const getKnobs = () => {
-  return {
-    value: text('value', '$5022'),
-    title: text('title', 'Total Earnings'),
-    colorMode: select('Color mode', [BigValueColorMode.Value, BigValueColorMode.Background], BigValueColorMode.Value),
-    graphMode: select('Graph mode', [BigValueGraphMode.Area, BigValueGraphMode.None], BigValueGraphMode.Area),
-    justifyMode: select('Justify', [BigValueJustifyMode.Auto, BigValueJustifyMode.Center], BigValueJustifyMode.Auto),
-    width: number('Width', 400, { range: true, max: 800, min: 200 }),
-    height: number('Height', 300, { range: true, max: 800, min: 200 }),
-    color: color('Value color', 'red'),
-    textMode: select(
-      'Text mode',
-      [BigValueTextMode.Auto, BigValueTextMode.Name, BigValueTextMode.ValueAndName, BigValueTextMode.None],
-      BigValueTextMode.Auto
-    ),
-  };
-};
-
+const NOOP_CONTROL = { control: { disable: true } };
```

Comment:
Now this is only visible for this story, would be hard to share it with other stories 😉

Reply:
So...exporting this variable automatically makes it a story of which is not what we need. I think it's a bit tricky to figure out how to reuse variables declared in a story elsewhere

Reply:
Maybe move it to here: https://github.com/grafana/grafana/blob/master/packages/grafana-ui/.storybook/preview.ts